src/lib.rs

```diff
@@ -86,13 +86,46 @@ pub mod store;
 pub mod tests;
 pub mod translate;
 
-pub use data::*;
-pub use executor::*;
-pub use parse_sql::*;
-pub use plan::*;
-pub use result::*;
-pub use store::*;
-pub use translate::*;
+pub mod prelude {
+    #[cfg(feature = "sled-storage")]
+    pub use crate::storages::SledStorage;
 
-pub use glue::Glue;
-pub use storages::*;
+    #[cfg(feature = "memory-storage")]
+    pub use crate::storages::MemoryStorage;
+
+    pub use crate::{
+        data::value::Value,
+        executor::{execute, Payload},
+        glue::Glue,
+        parse_sql::parse,
+        translate::translate,
```

Comment:
`parse` -> `translate` -> `plan` -> `execute`
This path above is generally expected routine if user does not use `Glue`.
Let's add `plan::plan` to prelude :)